File: Java/pagamento.java
import java.util.Locale;
import java.util.Scanner;

public class pagamento {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		String nome;
		double vlHora, pagamento;
		int horasTrab;

		System.out.print("Nome: ");
		nome = sc.nextLine();
		System.out.print("Valor por hora: ");
		vlHora = sc.nextDouble();
		System.out.print("Horas trabalhadas: ");
		horasTrab = sc.nextInt();

		pagamento = horasTrab * vlHora;

		System.out.printf("O pagamento para %s deve ser %.2f", nome, pagamento);

		sc.close();
	}
}

File: Python/pagamento.py
nome = str(input("Nome: "))
vl_hora = float(input("Valor por hora: "))
horas_trab = int(input("Horas trabalhadas: "))

pagamento = horas_trab * vl_hora
print(f"O pagamento para {nome} deve ser {pagamento:.2f}")